### ISSUE TITLE
No Sound playing movies

### ISSUE THREAD

**Issue by graemetgd** [clsid2/mpc-hc#2164]
Hi guys. I have not been able to figure this out but I have noticed that alot of Movies, dont have sound when playing through Media Player Classic. I also dont want to use VLC as Media Player has been my go to for YEARS. I am not sure if its something I have done regards codecs etc. But I cant figure it out. Sorry to be vague.

**Comment by clsid2** [comment]
Options > MIscellaneous > Reset

**Comment by graemetgd** [comment]
Hi there

Thank-you for still keeping Media Player classic going. To me it will always be my go to Player. Hence why I signed up here to say thanks and see if I could keep you motivated to keep doing the great work.

But sadly I Tried above. No luck. I am not sure if a Machine/PC restart is needed after that. But VLC seems to play all said movies-No issue sound wise. Media player does only play some. Tried a few now to test again.
